### ISSUE TITLE
Output Page Number to Extracted CSV

### ISSUE THREAD

**Issue by BallCity** [tabulapdf/tabula#565]
Hi,

Is it possible to output the Page Number from the source PDF to the resultant CSV (or whatever format one is outputting to) for every line? This would be very helpful from a quality control standpoint / splitting up PDFs that have a bunch of different table formats.

Thanks!

**Comment by jeremybmerrill** [comment]
No, this is not currently possible. One of Tabula's aims is to extract the table content as faithfully as we can, and, so, adding page numbers to the CSV that don't exist in the PDF would be out of scope.

However, the data JSON option could conceivably include this. It doesn't, at the moment. I'll mark this as an enhancement that we'll consider down the road.

**Comment by Shimakuchi** [comment]
Hi, any changes here? I have lets say 3 sections in one page to extract. So I need to know which 3 json object form a set. Insert a page number is one option. Or each page could be a json object which contains my 3 sections as nested objects.

I tried to build a workaround... The json diamension export contains the page number, but I found no referens to the json data export
